go.mod

```diff
@@ -17,15 +17,15 @@ require (
 	github.com/containerd/typeurl v1.0.3-0.20220422153119-7f6e6d160d67
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.2.0
-	github.com/containers/buildah v1.30.0
-	github.com/containers/common v0.52.0
+	github.com/containers/buildah v1.29.0
+	github.com/containers/common v0.51.0
 	github.com/containers/conmon v2.0.20+incompatible
 	github.com/containers/conmon-rs v0.4.0
-	github.com/containers/image/v5 v5.25.0
+	github.com/containers/image/v5 v5.24.0
 	github.com/containers/kubensmnt v1.2.0
 	github.com/containers/ocicrypt v1.1.7
-	github.com/containers/podman/v4 v4.5.2-0.20230706090613-813f1b53bf39
-	github.com/containers/storage v1.46.1
+	github.com/containers/podman/v4 v4.4.0-rc2
```

Comment:
maybe 4.4.1?

Reply:
That will bump c/storage to 1.45.6, is that OK?

Reply:
Related to:

- https://github.com/containers/podman/pull/20881

Reply:
@TomSweeneyRedHat, that would be fine. We will be updating containers/storage to **v1.45.6** anyway.